packages/gatsby-theme-store/src/sdk/buyButton/useBuyButton.ts

```diff
@@ -54,7 +70,7 @@ export const useBuyButton = ({
       return
     }
 
-    const isThisItem = e.data.items[0].id?.toString() === sku?.itemId
+    const isThisItem = e.data.items[0].skuId?.toString() === sku?.itemId
```

Comment:
Always e.data.items[0] will exist? If not, use the `?.` operator

Reply:
It always will. Because we send this event ourselves, and we make sure it always has one element.